### ISSUE TITLE
No capture devices are available 

### ISSUE THREAD

**Issue by Jiachengxiao** [cgsaxner/HoloLens2-Unity-ResearchModeStreamer#9]
Hi @cgsaxner 
Thank you for your great work! Actually, I successfully run the project for serveral times. However, I sometimes can't get the ahat data. Only the pv data is available. I check the debug output and the exception is as follows：
WinRT originate error - 0xC00DABE0 : 'No capture devices are available.'
Do you know how to fix this problem？

**Comment by cgsaxner** [comment]
Hi @Jiachengxiao,
glad to hear you are enjoying the project.

What you are describing seems to be a known bug, where the depth sensor stops delivering frames after the device has gone to standby. There is an [issue](https://github.com/microsoft/HoloLens2ForCV/issues/10) about this on the official HoloLens2ForCV repo, which was closed, although to the best of my knowledge, the bug is not yet fixed.

Unfortunately, I have not yet found a workaround for this yet :/ Only rebooting the HoloLens seems to work. I also set the _sleep settings_ to the maximum duration before the device goes to standby (30 minutes) so I don't have to reboot so often.
